LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) Andre Hofmeister and Contributors
+Copyright (c) 2019 - 2022 Andre Hofmeister and other authors
```

Comment:
I'd suggest removing the end year, to avoid moving that value every year
```suggestion
Copyright (c) 2019 Andre Hofmeister and other authors
```

Reply:
According to attorneys from my former firm, both years are required (or at least the highly recommend it). TBH I have no clue what is right 😅

Reply:
🤯 I followed this: https://choosealicense.com/licenses/mit/

I also considered using `Copyright (c) 2019 - current Andre Hofmeister and other authors`, although not sure if it's accepted by lawyers

Reply:
So we all aren't lawyers. In Java, we also use the years approach, let's do it for now like this, since we at least have one lawyer-backed data point by @HofmeisterAn .